components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.html

```diff
@@ -15,18 +15,57 @@
 </button>
 <chef-click-outside omit="dropdown-label" (clickOutside)="handleEscape()">
   <chef-dropdown class="dropdown" id="projects-filter-dropdown" [attr.visible]="dropdownActive" (keydown.esc)="handleEscape()">
-    <div class="dropdown-body">
-      <span class="optgroup-label">Projects</span>
-      <chef-checkbox
-        *ngFor="let option of editableOptions; index as i"
-        [attr.title]="option.label"
-        [checked]="option.checked"
-        (change)="handleOptionChange($event, i)"
-        (keydown.arrowup)="handleArrowUp($event)"
-        (keydown.arrowdown)="handleArrowDown($event)">{{ option.label }}</chef-checkbox>
-      <div class="dropdown-footer">
-        <chef-button primary [attr.disabled]="!optionsEdited" (click)="handleApplyClick()" id="projects-filter-apply-changes">Apply Changes</chef-button>
+
+      <div class="dropdown-body">
+
+        <div class="dropdown-content">
+          <!-- @TODO turn this into reusable Filter Input component -->
+          <input 
+            chefInput
+            type="text"
+            [(ngModel)]="filterValue"
```

Comment:
This should be revisited after tasks for upcoming sprint have been completed.  We would like to be removing these from the codebase, not adding.

Reply:
How will it be changed?